operations/mimir-mixin/config.libsonnet

```diff
@@ -51,6 +51,11 @@
     job_labels: [$._config.per_cluster_label, 'namespace', 'job'],
     cluster_labels: [$._config.per_cluster_label, 'namespace'],
 
+    // PromQL queries used to find clusters and namespaces with Mimir.
+    dropdown_job_query: 'cortex_build_info',  // Only used if singleBinary is true.
+    dropdown_cluster_query: 'cortex_build_info',
+    dropdown_namespace_query: 'cortex_build_info{%s=~"$cluster"}' % $._config.per_cluster_label,
```

Comment:
nit: if these share the same prefix, can they be grouped into a `dropdowns` object within `_config`?

Reply:
Good idea, thanks.